rails/project_feet_wet.md

```diff
@@ -106,7 +106,4 @@ In this chapter, you'll build a simple scaffolded app on Rails.  It won't be as
 * [Niño Mollaneda's solution](https://github.com/ninoM/toy_app)
 * [Jonathan Marks's solution](https://github.com/johnjmarks4/toy_app)
 * [Luján Fernaud's solution](https://github.com/lujanfernaud/rails-toy-app)
-
```

Comment:
Hi @anistor86,
we still need to keep the additional resources after you submit a solution